core/src/main/java/com/graphhopper/routing/util/BikeCommonFlagEncoder.java

```diff
@@ -151,10 +154,10 @@ protected BikeCommonFlagEncoder(int speedBits, double speedFactor, int maxTurnCo
         final int CYCLEWAY_SPEED = 18;  // Make sure cycleway and path use same speed value, see #634
         setHighwaySpeed("cycleway", CYCLEWAY_SPEED);
         setHighwaySpeed("path", 10);
```

Comment:
Narrator: They did not, in fact, make sure cycleway and path used the same speed value

I wonder why they ended up reversing this change https://github.com/graphhopper/graphhopper/issues/634

Reply:
It may have been a merge mistake.

It happened in this commit https://github.com/bikehopper/graphhopper/commit/85ef586dc6007521ce2e434ce4a2f40fe2657a31 which seems to be the merge for https://github.com/bikehopper/graphhopper/pull/635/files